src/DynamoCoreWpf/UI/Prompts/DynamoMessageBox.xaml

```diff
@@ -110,8 +110,21 @@
                            Grid.Row="2"
                            Margin="0,0,20,20"
                            LastChildFill="False">
+                    
+                    <Button Name="CancelButton"
+                            MinWidth="90"
```

Comment:
We had an issue somewhere saying German translation of cancel would take a lot of space making cancel button looks like cut. I suspect you may run into same issue here, our team has no solution yet. I think the task is https://jira.autodesk.com/browse/DYN-4404

Reply:
I'll take a look, but I've made it so the buttons now grow in width, see the screenshot above.

Reply:
@mjkkirschner Neat, then we may come back here when tackling the other one